<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>general_meeting</class>
 <widget class="QDialog" name="general_meeting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
     <widget class="QLabel" name="sponsor_label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Name of Sponsor:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="sponsor_input"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="meeting_point_label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Meeting Point Sheet:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QPushButton" name="meeting_upload">
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>verticalLayoutWidget_6</zorder>
   <zorder>meeting_point_label</zorder>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>200</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="point_sheet_label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Latest Point Sheet:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QPushButton" name="point_sheet_upload">
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>verticalLayoutWidget_5</zorder>
   <zorder>point_sheet_label</zorder>
  </widget>
  <widget class="QLabel" name="backLabel">
   <property name="geometry">
    <rect>
     <x>-230</x>
     <y>-210</y>
     <width>1131</width>
     <height>781</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="general_meeting_title">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>363</width>
     <height>78</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>General Meeting Points</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>331</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>NEXT</string>
   </property>
  </widget>
  <zorder>backLabel</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>verticalLayoutWidget_3</zorder>
  <zorder>verticalLayoutWidget_4</zorder>
  <zorder>general_meeting_title</zorder>
  <zorder>next_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
